### DISCUSSION TITLE
Slint v1.5 Android: How to create multiple android activities and jump between them?

### DISCUSSION THREAD

**Discussion by heng30** [slint-ui/slint#5027] [Q&A]
Adroid app is base on activities, different activity server for different functionality. I didn't find an example can create a multiple activities. If I press a button, can jump to another activity. which makes the app more like a mobile app not a desktop app.

**Comment by ogoffart** [comment]
I don't know if that's possible with the native activity.
I guess you'll have to reach into java with JNI for that.
Let me know if you find a way to do it and if there is something we should do in Slint to make it easier.
